virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -79,13 +79,18 @@ The *Overview* page displays resources, metrics, migration progress, and cluster
 |Status of live migrations.
 
 |xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings_virt-web-console-overview[*Settings* tab]
-|The *Settings* tab contains the *Cluster* tab and the *User* tab.
+|The *Settings* tab contains the *Cluster* tab, *User* tab, and *Preview features* tab.
 
 |*Settings* -> xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[*Cluster* tab]
 |{VirtProductName} version, update status, live migration, templates project, preview features, and load balancer service settings.
 
 |*Settings* -> xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-user_virt-web-console-overview[*User* tab]
 |Authorized SSH keys, user permissions, and welcome information settings.
+
+|*Settings* -> *Preview features*
+|Enable select link:https://access.redhat.com/support/offerings/techpreview/[preview features] in the UI. Features in this tab change frequently.
```

Comment:
You refer to this as a tab in the sentence after it so I think this should be, `[*Preview features*] in the UI.`

Reply:
Leaving this as-is. This "preview features" is referring to preview features in general and not to the tab itself.